domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1022,438 +1074,562 @@ private void setUpUserDBForTest(TestContext testContext, PostgresClient pg) {
       return ResponseImpl.respond204();
     }
+
     public static Response respond400WithTextPlain(Object entity) {
       return ResponseImpl.respond400WithTextPlain(entity);
     }
+
     public static Response respond500WithTextPlain(Object entity) {
       return ResponseImpl.respond500WithTextPlain(entity);
     }
-  };
+  }
+
+  ;
 
   static class ResponseWithout204 extends ResponseDelegate {
+
     private ResponseWithout204(Response response) {
       super(response);
     }
+
     public static Response respond200WithApplicationJson(User entity) {
       return ResponseImpl.respond200WithApplicationJson(entity);
     }
+
     public static class HeadersFor201 extends ResponseImpl.HeadersFor201 {
+
     }
+
     public static HeadersFor201 headersFor201() {
       return new HeadersFor201();
     }
+
     public static Response respond201WithApplicationJson(Object entity, HeadersFor201 headers) {
       return ResponseImpl.respond201WithApplicationJson(entity, headers);
     }
+
     public static Response respond400WithTextPlain(Object entity) {
       return ResponseImpl.respond400WithTextPlain(entity);
     }
+
     public static Response respond500WithTextPlain(Object entity) {
       return ResponseImpl.respond500WithTextPlain(entity);
     }
-  };
+  }
+
+  ;
 
   static class ResponseWithout400 extends ResponseDelegate {
+
     private ResponseWithout400(Response response) {
       super(response);
     }
+
     public static Response respond200WithApplicationJson(User entity) {
       return ResponseImpl.respond200WithApplicationJson(entity);
     }
+
     public static class HeadersFor201 extends ResponseImpl.HeadersFor201 {
+
     }
+
     public static HeadersFor201 headersFor201() {
       return new HeadersFor201();
     }
+
     public static Response respond201WithApplicationJson(Object entity, HeadersFor201 headers) {
       return ResponseImpl.respond201WithApplicationJson(entity, headers);
     }
+
     public static Response respond204() {
       return ResponseImpl.respond204();
     }
+
     public static Response respond500WithTextPlain(Object entity) {
       return ResponseImpl.respond500WithTextPlain(entity);
     }
-  };
+  }
+
+  ;
 
   static class ResponseWithout500 extends ResponseDelegate {
+
     private ResponseWithout500(Response response) {
       super(response);
     }
+
     public static Response respond200WithApplicationJson(User entity) {
       return ResponseImpl.respond200WithApplicationJson(entity);
     }
+
     public static class HeadersFor201 extends ResponseImpl.HeadersFor201 {
+
     }
+
     public static HeadersFor201 headersFor201() {
       return new HeadersFor201();
     }
+
     public static Response respond201WithApplicationJson(Object entity, HeadersFor201 headers) {
       return ResponseImpl.respond201WithApplicationJson(entity, headers);
     }
+
     public static Response respond204() {
       return ResponseImpl.respond204();
     }
+
```

Comment:
860 lines were changed in this file.
Most of them are not related to the counts issue.
Please create a separate pull request for the whitespace and line wrapping changes.
Without the whitespace and line wrapping changes this pull request can more easily been code reviewed and been backported.

Reply:
There are still 515 lines with whitespace/line wrapping changes unrelated to this counts issue in PgUtilIT.

Reply:
Still there.